addons/sys_core/stringtable.xml

```diff
@@ -278,49 +182,13 @@
             <Portuguese>Mudar Idioma (Babel)</Portuguese>
         </Key>
         <Key ID="STR_ACRE_sys_core_RadioLeftEar">
-            <English>Radio Left Ear</English>
-            <Spanish>Radio auricular izquierdo</Spanish>
-            <Czech>Rádio do levého ucha</Czech>
-            <German>Linker Ohrhörer</German>
-            <Japanese>音声を左側から</Japanese>
-            <Italian>Imposta Auricolare Sinistro</Italian>
-            <French>Radio oreille gauche</French>
-            <Korean>왼쪽 이어폰</Korean>
-            <Polish>Odsłuch tylko na lewym uchu</Polish>
-            <Russian>Радио левое ухо</Russian>
-            <Chinese>用左耳收聽無線電</Chinese>
-            <Chinesesimp>用左耳收听无线电</Chinesesimp>
-            <Portuguese>Rádio Orelha Esquerda</Portuguese>
+            <English>Radio Left Ear (Active Radio)</English>
```

Comment:
I think you should stick to "selected" throughout - above you use "activate" to mean press the push-to-talk.

I'm not keen on the Title Casification Of These - `Radio left ear (selected radio)` looks neater to me.

Reply:
I use "activate" in sense of activating the PTT button on the radio, yes.

"Selected" is longer than "Active", I don't really see the difference otherwise.